units/build.gradle.kts

```diff
@@ -10,9 +10,6 @@ plugins {
     id("detekt-convention")
 }
 
-group = "dev.icerock.moko"
-version = libs.versions.mokoUnitsVersion.get()
-
 android.buildFeatures.dataBinding = true
```

Comment:
databinding should be removed here. and kapt too, and all from databinding

Reply:
removed